src/base/text/funcstring.cpp

```diff
@@ -22,15 +23,18 @@ FuncString::FuncString(std::string code, bool throwOnError)
 	}
 
 	if (parts[1].empty() || parts[1].back() != ')') {
-		if (!throwOnError) return;
+		if (!throwOnError) {
```

Comment:
please check if changed functionality does not cause problem where it is used

Reply:
I reverted this modification